### PR TITLE
patch

### PR DESCRIPTION
**Pull request by viktoriias**
## Essential Checklist

- [ ] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [ ] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [ ] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [ ] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [ ] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Comment by anandwana001** [comment]
@viktoriias Is this PR specific to any issue?

**Comment by viktoriias** [comment]
> @viktoriias Is this PR specific to any issue?

I'll link it to the issue if I get it to work

File: app/src/main/res/drawable/circular_solid_color_primary_32dp.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+  android:shape="rectangle">
+  <solid android:color="@color/colorPrimary" />
+  <corners android:radius="16dp" />
+  <size
+    android:width="32dp"
+    android:height="32dp" />
+</shape>
```
Comment (rt4914) [inline app/src/main/res/drawable/circular_solid_color_primary_32dp.xml:9]: EOF is missing, add one blank line at the end.

File: app/src/main/res/drawable/circular_stroke_2dp_color_primary_32dp.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  android:shape="rectangle">
+  <solid android:color="@color/white" />
+  <stroke
+    android:width="2dp"
+    android:color="@color/colorPrimary" />
+  <corners android:radius="16dp" />
+  <size
+    android:width="32dp"
+    android:height="32dp" />
+</shape>
```
Comment (rt4914) [inline app/src/main/res/drawable/circular_stroke_2dp_color_primary_32dp.xml:12]: EOF is missing, add one blank line at the end.

File: app/src/main/res/layout/test_image_view_bindable_adapter_activity.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <ImageView
+      android:id="@+id/image_view_for_data_binding"
+      android:layout_width="32dp"
+      android:layout_height="32dp"/>
+
+</layout>
```
Comment (rt4914) [inline app/src/main/res/layout/test_image_view_bindable_adapter_activity.xml:9]: EOF is missing, add one blank line at the end.

**Review by rt4914 — CHANGES_REQUESTED** [review]
@viktoriias Can you please merge PR with latest develop that might pass the CI checks. (FYI: there are some flaky tests currently on CI)

**Pull request closed without merging**